src/renderer/components/InvitationsNew/index.vue

```diff
@@ -0,0 +1,188 @@
+<template >
+
+<el-row>
+    <el-dialog
+      title="Scan Invitation"
+      :visible.sync="QRDialogVisible"
+      width="500">
+      <div style="margin-left:auto;margin-right:auto;">
+        <qrcode v-bind:value="QRDialogURL" :options="{ width: 400 }"></qrcode>
+      </div>
+      <span slot="footer" class="dialog-footer">
+        <el-button type="primary" @click="QRDialogVisible = false">Close</el-button>
+      </span>
+    </el-dialog>
+
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand" href="#">Invitations</a>
+      <el-button
+        type="primary"
+        icon="el-icon-refresh"
+        @click="fetch_invitations_v1"></el-button>
+  </nav>
+  <el-collapse v-model="expanded_items">
+      <ul class="list">
+        <el-collapse-item
+          v-for="i in invitations_v1"
+          v-bind:title="i.alias"
+          :name="i.alias"
+          :key="i.id">
+          <el-row :key="i.id">
+            <p>Auto Accept: {{i.auto_accept}}</p>
+            <p>Role: {{i.role}}</p>
+            <p>Multi-use: {{i.multi_use}}</p>
+            <p>Created: {{i.created_date}}</p>
+            <el-button type="primary" @click="copyURL(i.invitation_url)">Copy URL</el-button>
+            <el-button type="primary" @click="presentQR(i.invitation_url)">Scan QR</el-button>
+            <div>
+              <vue-json-pretty
+                :deep=0
+                :data="i">
+              </vue-json-pretty>
+            </div>
+          </el-row>
+        </el-collapse-item>
+      </ul>
+    </el-collapse>
+
+  <el-divider></el-divider>
+
+  <p>Create Invitations:</p>
+  <el-form :inline="false" label-width="120px">
+    <el-form-item label="Alias:">
+      <el-input v-model="invite_alias_form" style="width:200px;"> </el-input>
+      <i>Alias used in your list of invitations.</i>
+    </el-form-item>
+    <el-form-item label="Label:">
+      <el-input v-model="invite_label_form" style="width:200px;"> </el-input>
+      <i>The label is presented in the invitation to the recipient.</i>
```

Comment:
Wording is a bit awkward
```suggestion
      <i>The label presented in the invitation to the recipient.</i>
```